src/session/hooks.tsx

```diff
@@ -22,5 +24,7 @@ export const useSessionField = (fieldPath, defaultValue = null) => {
 
 export const useSetSession = () => {
   const dispatch = useDispatch();
-  return useCallback((...args) => dispatch(setSession(...args)), []);
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  return useCallback((...args: unknown[]) => dispatch(setSession(...args)), []);
```

Comment:
Big question on this part. Eventually, can't find a way to type this part properly without changing an API. Would appreciate ideas @iamuchejude @peterhass